models/vyos/bgp_address_family/vyos_bgp_address_family.yaml

```diff
@@ -0,0 +1,210 @@
+module: vyos_bgp_family
+version_added: 2.0.1
+short_description: BGP Address Family Resource Module.
+description: 
+- This module manages BGP address family configuration of interfaces on devices running VYOS.
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+options:
+  config:
+    description: A dict of BGP global configuration for interfaces.
+    type: dict
+    suboptions:
+      as_number:
+        description:
+        - AS number.
+        type: int
+      address_family:
+        description: BGP address-family parameters.
+        type: list
+        elements: dict
+        suboptions:
+          afi:
+            description: BGP address family settings.
+            type: str
+            choices: ['ipv4_unicast', 'ipv6_unicast']
```

Comment:
Looks like VyOS supports SAFI. Can we have a separate key for it then like we do for some of the other platforms?

Reply:
Do you mean 'unicast' ? ipv4-unicast is represented as a single parameter in vyos. Same is case with ipv6-unicast